source/client/events.js

```diff
@@ -1,54 +1,70 @@
-Space.messaging.define Space.messaging.Event, 'Space.accountsUi', {
+Space.messaging.define(Space.messaging.Event, 'Space.accountsUi', {
 
-  # SIGNUP
+  // SIGNUP
 
   SignupRequested: {
     username: Match.OneOf(Username, null),
     email: Match.OneOf(EmailAddress, null),
     password: Match.OneOf(Password, null)
-  }
+  },
 
   SignupInitiated: {
     signupId: Guid
-  }
+  },
 
   SignupRetried: {
     signupId: Guid,
-  }
+  },
 
   SignupFailed: {
     signupId: Guid,
     error: String
-  }
+  },
 
   SignupCompleted: {
     signupId: Guid
-  }
+  },
 
-  # LOGIN
+  // LOGIN
 
   LoginRequested: {
-    user: String
+    user: String,
     password: Password
-  }
+  },
 
   LoginInitiated: {
     user: String
-  }
+  },
 
   LoginFailed: {
-    user: String
+    user: String,
     error: Meteor.Error
-  }
+  },
 
   LoginSucceeded: {
     user: String
-  }
+  },
+
+  // LOGIN WITH GOOGLE
+  LoginWithGoogleRequested: {
+
+  },
```

Comment:
if there are not properties, just keep it compact: `LoginWithGoogleRequested: {}`